service/storage_service.cc

```diff
@@ -1526,7 +1526,7 @@ future<> storage_service::await_tablets_rebuilt(raft::server_id replaced_id) {
     };
```

Comment:
` storage_service: use conditional_vacriable::when in co-routines consistently`

typo in commit message